Tasks/PackerBuildV0/Tests/L0LinuxsBuiltinTemplateAdditionalParameters.ts

```diff
@@ -0,0 +1,141 @@
+import ma = require('vsts-task-lib/mock-answer');
+import tmrm = require('vsts-task-lib/mock-run');
+import path = require('path');
+
```

Comment:
Isn't this file already there? How come this is being added?

Reply:
I added the duplicate file by mistake when resolving merge conflicts, the only difference was a s in the file name. Have removed the duplicate file, thanks.